src/lib/axl-explorations/src/components/games/LetterHuntGameCore.tsx

```
@@ -787,20 +787,19 @@ export function LetterHuntGameCore({
-                  ? 'bg-blue-100 cursor-pointer hover:bg-blue-200 hover:scale-110 ring-4 ring-blue-400 ring-opacity-50 animate-pulse'
-                  : mode === 'preview' && demoStep === 'instruction1'
+              className={`inline-block p-0.5 sm:p-1 md:p-1.5 rounded-lg transition-colors ${mode === 'preview' && demoStep === 'waitForSpeaker' && !hasClickedSpeaker
+                ? 'bg-blue-100 cursor-pointer hover:bg-blue-200 hover:scale-110 ring-4 ring-blue-400 ring-opacity-50 animate-pulse'
+                : mode === 'preview' && demoStep === 'instruction1'
                   ? 'bg-gray-100 cursor-not-allowed opacity-50'
                   : 'bg-blue-100 cursor-pointer hover:bg-blue-200'
-              }`}
+                }`}
               onClick={handleSpeakerClick}
               tabIndex={mode === 'preview' && demoStep === 'waitForSpeaker' ? 0 : -1}
             >
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)